tests/test_operator.py

```diff
@@ -232,6 +234,48 @@ def test_dimension_size_override(self, nt=100):
         op(a=a, time=5)
         assert(np.allclose(a.data[0], 4.))
 
+    def test_override_composite_data(self):
+        dimensions = (100, 100)
+        time_order = 2
+        space_order = 4
+        solver = setup(dimensions=dimensions, time_order=time_order,
```

Comment:
I dont think we need the full complexity of an example setup to test this. In fact it makes the test really unreadable, since it presumes knowledge of an acoustic example, when all we want to test is whether `PointData.coordinates` is used correctly. So, could we not instead do something like (pseudo-code):
```
src1 = PointData(name='src1', coordinates=(1, 1))
src2 = PointData(name='src2', coordinates=(2, 2))
op = Operator(src1.inject(1.))
op(src1=src2)
```
and then test where the `1` values were injected in a grid of otherwise `0`s?